rest-api/src/main/kotlin/fund/cyber/markets/api/rest/controller/RawDataController.kt

```diff
@@ -0,0 +1,52 @@
+package fund.cyber.markets.api.rest.controller
+
+import fund.cyber.markets.api.rest.configuration.RestApiConfiguration
+import fund.cyber.markets.common.model.OrderBook
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.http.ResponseEntity.ok
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.client.HttpClientErrorException
+import org.springframework.web.client.RestTemplate
+import org.springframework.web.util.UriComponentsBuilder
+import reactor.core.publisher.Mono
+import reactor.core.publisher.toMono
+
+
+const val ORDERBOOK_PATH = "/orderbook"
+
+@RestController
+class RawDataController {
+
+    @Autowired
+    private lateinit var configuration: RestApiConfiguration
```

Comment:
Don't need to autowire configuration here. Inject property itself or create bean containing all needed properties and inject it